src/reanimated2/layoutReanimation/web/animationParser.ts

```diff
@@ -0,0 +1,62 @@
+export interface ReanimatedWebTransformProperties {
+  translateX?: string;
+  translateY?: string;
+  rotate?: string;
+  rotateX?: string;
+  rotateY?: string;
+  scale?: number | string;
+  scaleX?: number;
+  scaleY?: number;
+  perspective?: string;
+  skew?: string;
+  skewX?: string;
+}
+
+interface AnimationStyle {
+  opacity?: number;
+  transform?: ReanimatedWebTransformProperties[];
+}
+
+export interface AnimationData {
+  name: string;
+  style: Record<number, AnimationStyle>;
+  duration: number;
+}
```

Comment:
```suggestion
}

```

Reply:
Done in [5aafc4d](https://github.com/software-mansion/react-native-reanimated/pull/4805/commits/5aafc4d8a2c09e39ccdf76d490e7446b27320ae2).